frontend/src/scenes/insights/InsightsTable/InsightsTable.tsx

```diff
@@ -164,10 +164,7 @@ export function InsightsTable({ isLegend = true, showTotalCount = false }: Insig
                         filters.display === ACTIONS_PIE_CHART)
                 ) {
                     // Special handling because `count` will contain the last amount, instead of the cumulative sum.
```

Comment:
@paolodamico Do you know when count shows last amount over cumulative sum? It seems like it's passing cumulative sum now, but want to double check 😄

Reply:
Can't remember from the top of my head when this happens, tried a bunch of different things and couldn't find this happening so I think it's safe to remove. Maybe worth just checking with Core Analytics as a precaution.